src/main/kotlin/org/jitsi/nlj/RtpEncodingDesc.kt

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.nlj
+
+import org.jitsi.nlj.rtp.SsrcAssociationType
+import org.jitsi.nlj.rtp.VideoRtpPacket
+import org.jitsi.nlj.stats.NodeStatsBlock
+
+/**
+ * Keeps track of information specific to an encoded RTP stream.
```

Comment:
So, is this 1:1 with an RTP stream (ssrc)?  If so, is there a reason to have a term "encoding" if it's 1:1 with stream?

Reply:
It's an RTP encoded stream plus its associated secondary SSRCs, which is a concept that RFC 7656 didn't specifically address. I've updated the docs accordingly.